pkg/printer/printer.go

```diff
@@ -189,3 +194,11 @@ func (p *Printer) printPassedYamlValidation() {
 func (p *Printer) printSkippedPolicyCheck() {
 	p.printInColor("[?] Policy check didn’t run for this file\n", p.theme.Colors.White)
 }
+
+func (p *Printer) getStringOrNotAvailable(str string) string {
+	if str == "" {
```

Comment:
Will this work if `str` is `undefined`?
I think we can address that as well

Reply:
in golang, string cannot be nil (only a pointer to a string can)